Tutorials/whatiswot/12-Building_Blocks/12-Building-Blocks.vtt

```diff
@@ -0,0 +1,197 @@
+After the introduction, WoT Discovery defines a set of second-stage "exploration" mechanisms that provide actual access to metadata.
+
+00:03:26.646 --> 00:03:30.634 
+This can be done in 2 ways: a single TD can
+
+00:03:30.634 --> 00:03:34.621 
+be retrieved or a link for TD directories becomes available.
+
+00:03:34.621 --> 00:03:39.596 
+It is a database with a REST API to access TDs.
+
+00:03:39.596 --> 00:03:50.171 
+Thing Description Directory service provides mechanisms to register, and retrieve WoT metadata and TDs after authentication and authorization.
+
+00:03:50.171 --> 00:03:56.059 
+Thus we can summarize the Discovery mechanism like this.
+
+00:03:56.059 --> 00:04:00.021 
+The third building block is WoT Binding Templates.
+
+00:04:00.021 --> 00:04:08.290 
+The IoT uses a variety of protocols for accessing devices because no single protocol is enough to cover the different
+
+00:04:08.290 --> 00:04:16.559 
+use cases and requirements. The challenge of the WoT is to enable interactions of a variety of different IoT platforms.
+
+00:04:16.559 --> 00:04:23.016 
+The Binding Templates address an application client to use the TD so that they can extract
+
+00:04:23.016 --> 00:04:29.474 
+metadata of the protocols such as HTTP and MQTT etc., payload formats such as JSON and
+
+00:04:29.474 --> 00:04:35.931 
+CBOR, and their usage in an IoT framework context such as ECHONET and OPC.
+
+00:04:35.931 --> 00:04:42.403 
+In general, Binding Templates provides a collection of blueprints that give guidance on how to
+
+00:04:42.403 --> 00:04:48.874 
+interact with different Things that use different transport protocols, and content types or that are
+
+00:04:48.874 --> 00:04:55.346 
+different IoT platforms or standards that use certain combinations of transport protocols and content types.
+
+00:04:55.346 --> 00:04:58.996 
+Now we will introduce the WoT profile.
+
+00:04:58.996 --> 00:05:06.183 
+By nature, TD is flexible and able to describe anything without any limitations.
+
+00:05:06.183 --> 00:05:14.940 
+However, in some cases let's say you are trying to communicate with your smart bulb with your smartphone,
+
+00:05:14.940 --> 00:05:23.696 
+the lack of limitation of the TDs causes an unnecessary implementation load to communicate with every Thing.
+
+00:05:23.696 --> 00:05:29.021 
+and while implementing a thing profile makes it way easier with providing guidance.
+
+00:05:29.021 --> 00:05:34.346 
+For instance, developers don’t have to think about which protocols to choose.
+
+00:05:34.346 --> 00:05:43.353 
+Thus, in those cases, Profile limits the capabilities of the TDs which enables out of box interoperability among things and
+
+00:05:43.353 --> 00:05:52.359 
+devices. And make it possible to communicate with only minor configuration operations such as entering a key or IP address.
+
+00:05:52.359 --> 00:06:00.275 
+Lastly, we will introduce WoT Scripting API, which is an optional building block of W3C WoT.
```

Comment:
w3c pronunciation seems off. We had agreed on one no?

Reply:
I have taken the same pronunciation from the 10th video's beginning, so it's the same that we were using before. Should we change it?

Reply:
I have checked https://www.w3.org/2023/09/TPAC/demos/what-is-wot.html and it seems very similar. Just a small confusion on my side :)